packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -225,242 +235,273 @@ contract Initializer_Test is Bridge_Initializer {
-                initializedSlotVal: deploy.loadInitializedSlot("OptimismMintableERC20Factory")
+                initCalldata: abi.encodeCall(l1OptimismMintableERC20Factory.initialize, (address(l1StandardBridge)))
             })
         );
         // OptimismMintableERC20FactoryProxy
         contracts.push(
             InitializeableContract({
+                name: "OptimismMintableERC20FactoryProxy",
                 target: address(l1OptimismMintableERC20Factory),
-                initCalldata: abi.encodeCall(l1OptimismMintableERC20Factory.initialize, (address(l1StandardBridge))),
-                initializedSlotVal: deploy.loadInitializedSlot("OptimismMintableERC20FactoryProxy")
+                initCalldata: abi.encodeCall(l1OptimismMintableERC20Factory.initialize, (address(l1StandardBridge)))
             })
         );
         // DataAvailabilityChallengeImpl
         contracts.push(
             InitializeableContract({
+                name: "DataAvailabilityChallenge",
                 target: deploy.mustGetAddress("DataAvailabilityChallenge"),
-                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0)),
-                initializedSlotVal: deploy.loadInitializedSlot("DataAvailabilityChallenge")
+                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0))
             })
         );
         // DataAvailabilityChallengeProxy
         contracts.push(
             InitializeableContract({
+                name: "DataAvailabilityChallengeProxy",
                 target: address(dataAvailabilityChallenge),
-                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0)),
-                initializedSlotVal: deploy.loadInitializedSlot("DataAvailabilityChallengeProxy")
+                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0))
+            })
+        );
+        // AnchorStateRegistry
+        contracts.push(
+            InitializeableContract({
+                name: "AnchorStateRegistry",
+                target: address(anchorStateRegistry),
+                initCalldata: abi.encodeCall(
+                    anchorStateRegistry.initialize,
+                    (new AnchorStateRegistry.StartingAnchorRoot[](1), SuperchainConfig(address(0)))
+                )
             })
         );
+        // AnchorStateRegistryProxy
+        contracts.push(
+            InitializeableContract({
+                name: "AnchorStateRegistryProxy",
+                target: address(anchorStateRegistry),
+                initCalldata: abi.encodeCall(
+                    anchorStateRegistry.initialize,
+                    (new AnchorStateRegistry.StartingAnchorRoot[](1), SuperchainConfig(address(0)))
+                )
+            })
+        );
+
+        // Nicknamed contracts.
+        nicknames["OptimismPortal2Proxy"] = "OptimismPortalProxy";
```

Comment:
Ah ok, so nicknames is just for this and should be deleted once the two portals are merged?

Reply:
Yes, basically. Mostly the nicknames thing is just for any cases where two contracts use the same proxy.